### ISSUE TITLE
don't use BASE and empty prefix; precise ontology URL; use turtle `@prefix` syntax

### ISSUE THREAD

**Issue by VladimirAlexiev** [opengeospatial/ogc-geosparql#474]
The ontology https://opengeospatial.github.io/ogc-geosparql/geosparql11/geo.ttl has this preamble:
```
BASE <http://www.opengis.net/ont/geosparql>

PREFIX : <http://www.opengis.net/ont/geosparql#>
PREFIX dcterms: <http://purl.org/dc/terms/>
PREFIX owl: <http://www.w3.org/2002/07/owl#>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX sdo: <https://schema.org/>
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
PREFIX spec11: <http://www.opengis.net/spec/geosparql/1.1/specification.html#> # TODO: use PID IRI
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
```
The ontology URL is
```ttl
<http://www.opengis.net/ont/geosparql> 
	a owl:Ontology ;
	dcterms:title "GeoSPARQL Ontology" ;
```
And then each term has `rdfs:isDefinedBy :` .

Suggestions:
- there are no relative URLs in the file, so don't use `BASE`
- don't use an empty prefix. Many ontologies do that (including SKOS) but I think it's a bad practice. Many repos (including GraphDB) record incoming Turtle prefixes in their namespace registries, but there can only be one empty prefix, so it will be appropriated by the first ontology to be loaded. 
- do declare the preferred prefix and namespace using VANN. According to prefix.cc (https://prefix.cc/geo), the most popular prefix for GeoSPARQL is `geo:`
- use the same URL in the ontology declaration and in `rdfs:isDefinedBy`. Currently the two differ since the latter has an extra `#` appended. This means that when a web crawler stumbles upon a GeoSPARQL term, it cannot discover the whole GeoSPARQL ontology.
- use the usual Turtle syntax `@prefix`. It is true that Turtle now also allows the SPARQL syntax `PREFIX` but IMHO using the usual syntax makes the ontology slightly more interoperable.

In summary, I propose to change it as follows:

```ttl
@prefix geo: <http://www.opengis.net/ont/geosparql#>.
@prefix vann: <http://purl.org/vocab/vann/>.
@prefix dcterms: <http://purl.org/dc/terms/>.
...

geo:
	a owl:Ontology ;
	dcterms:title "GeoSPARQL Ontology" ;
	vann: preferredNamespacePrefix "geo";
	vann:preferredNamespaceUri "http://www.opengis.net/ont/geosparql#";
...
<term> rdfs:isDefinedBy geo: ;
```

**Comment by VladimirAlexiev** [comment]
Note: if you change the ontolory URL to `http://www.opengis.net/ont/geosparql#`,
the same change will be needed in https://opengeospatial.github.io/ogc-geosparql/geosparql11/profile.ttl

**Comment by situx** [comment]
Thanks for the suggestions.
We will discuss it in our next meeting, which will be on January 10th, and then issue a pull request for the agreed-upon fixes.
